test/test_cuda.py

```diff
@@ -3057,7 +3057,7 @@ def test_autocast_rnn(self):
                 # Autocast wrapper requires at::_cudnn_rnn is autograd-exposed.  This check can't guarantee
                 # at::_cudnn_rnn is autograd-exposed, but if it fires, it indicates some funny business has
                 # occurred and we should double check that at::_cudnn_rnn remains autograd-exposed.
-                self.assertEqual(out.grad_fn.name(), "CudnnRnnBackward0")
+                self.assertEqual(out.grad_fn.name(), "CudnnRnnBackward0" if not TEST_WITH_ROCM else "MiopenRnnBackward0")
```

Comment:
```suggestion
                self.assertEqual(True, out.grad_fn.name() in ("CudnnRnnBackward0", "MiopenRnnBackward0"))
```
I haven't tested the above suggestion, but I'm curious what you think?  There have been a few tests lately with exact-string name matching like this one, and I prefer to not add more uses of TEST_WITH_ROCM when possible.

Reply:
I like that suggestion. Feels more "pythonic" and I agree with your reasoning.